app/src/main/java/com/riyas/myapplication/login_validation.kt

```diff
@@ -0,0 +1,36 @@
+class LoginValidator {
+
+    fun validateEmail(email: String): String {
```

Comment:
i dont think this is a good name . it has to be IsValidEmail() and should not return string but a boolean